spec/controllers/redirections_controller_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "rails_helper"
+
+describe RedirectionsController do
```

Comment:
eeeeinteresting. I did not know this was a thing now. I certainly don't like money patching. Perhaps I'll give this a shot on this project.